identity-aware-proxy/IAPClient/Main.cs

```diff
@@ -0,0 +1,64 @@
+﻿/*
+Copyright 2018 Google Inc
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+using System;
+using CommandLine;
+
+namespace GoogleCloudSamples
+{
+    class Options
+    {
+        const string IAP_CLIENT_ID = "YOUR-IAP-CLIENT-ID";

```

Comment:
Maybe worth putting a link that explains how to extract it from the portal

Reply:
Done.